src/App.js

```diff
@@ -69,8 +32,8 @@ export default function App() {
           element={
             <CreepypastaPage
               handleBookmarkClick={handleBookmarkClick}
```

Comment:
Since you are passing the function, it would be correct to use the prefix "on" after convention.